test/integration/verify/controls/aws_ecr.rb

```diff
@@ -0,0 +1,15 @@
+title 'Test single AWS ECR'
+
+aws_ecr_name = "aws-ecr-name-fhxglcpleahkwukwlzdyywblu"
+#aws_ecr_name = attribute(:aws_ecr_name, default: '', description: 'The AWS ECR name.')
+
+control 'aws-ecr-1.0' do
+
+  impact 1.0
+  title 'Ensure AWS ECR has the correct properties.'
+
+  describe aws_ecr(repository_name: aws_ecr_name) do
+    it                       { should exist }
+    its ('repository_name')  { should eq aws_ecr_name }
```

Comment:
Some controls around the `images` would be nice here.

Reply:
We currently don't have a good mechanism for pushing container images to ECR that would fit well into CI. We can revisit this later